pkg/utils/testutils.go

```diff
@@ -22,11 +22,25 @@ import (
 	"time"
 )
 
-const charset = "abcdefghijklmnopqrstuvwxyz" + "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+const alphabets = "abcdefghijklmnopqrstuvwxyz" + "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+const digits = "0123456789"
 
 var seededRand *rand.Rand = rand.New(rand.NewSource(time.Now().UnixNano()))
 
-func RandomStringWithCharset(length int) string {
+func GetRandomString(length int, stringType string) string {
+	var charset string
+	switch stringType {
+	case "alpha":
+		charset = alphabets
+	case "numeric":
+		charset = digits
+	case "alphanumeric":
```

Comment:
nit: use an enum instead of strings so that we get a compile-time error if there's a typo